docs/resiliency/index.asciidoc

```diff
@@ -64,6 +64,22 @@ framework. As the Jepsen tests evolve, we will continue porting new scenarios th
 all new scenarios and will report issues that we find on this page and in our GitHub repository.
 
 [float]
+=== Repeated network partitions can cause cluster state updates to be lost (STATUS: ONGOING)
+
+During a networking partition, cluster states updates (like mapping changes or shard assignments)
+are committed if a majority of the masters node received the update correctly. This means that the current master has access
+to enough nodes in the cluster to continue to operate correctly. When the network partition heals, the isolated nodes catch
+up with the current state and get the changes they couldn't receive before. However, if a second partition happens while the cluster
```

Comment:
get the changes they couldn't receive before -> receive the previously missed changes.